src/redux/reducers/rootReducer.js

```diff
@@ -0,0 +1,13 @@
+export const initialState = {
+    userIsLogin: false
+};
+
+export function rootReducer(state = initialState, action) {
+  switch (action.type) {
+    case 'SET_USER':
+      return action.userIsLogin;
```

Comment:
it will mutate your state as you set the value by link. 
```suggestion
      return {
          ...state,
          userIsLogin: action.userIsLogin
      }
```

Reply:
and prefer to use isLoggedIn var name